assets/panel/debugger.properties

```diff
@@ -452,7 +452,8 @@ editor.singleResult=1 result
 
 # LOCALIZATION NOTE (editor.noResults): Editor Search bar message
 # for when no results found.
-editor.noResults=No results found
+editor.noResults=No results
```

Comment:
Is this string still used after your changes? If it isn't, you should remove it from the file entirely, and update the localization note to reference the new string instead.